requirements.txt

```diff
@@ -12,7 +12,7 @@ django-braces
 django-autofixture
```

Comment:
Also , please remove this file too and create a new PR for updating the dependencies.